### DISCUSSION TITLE
Not starting on startup Win11 24 H2

### DISCUSSION THREAD

**Discussion by Conza89** [rocksdanister/lively#2405] [Q&A]
Since going to Win11 24 H2 it won't start on its own without any sort of adjustment. If I change DPI, refresh rate, etc, somehow that is enough for it to start running. It feels like it's being prevented from running somehow, no idea why not sure what's different.

Turning Windows HDR on then off, also does this.

**Comment by Conza89** [comment]
I have uninstalled and re-installed using the Microsoft Store version and have had no success yet. My work around of adjusting display DPI instantly triggers the wallpaper.

**Reply by rocksdanister** [reply]
🤔I didn't get 24H2 stable on my system yet, will need to take a look.

**Reply by Conza89** [reply]
No problems, it is available on Release preview for Windows Insiders. If there's any specific testing I can assist with, I'll do my best to help.

**Reply by rocksdanister** [reply]
Some users have reported turning HDR on and off fixed the issue: Windows + Alt + B.
